code/src/components/ThoughtsForm.js

```diff
@@ -0,0 +1,33 @@
+import React from "react";
+import { HeartIcon } from "./HeartIcon";
+
+export const ThoughtsForm = ({ onFormSubmit, setNewMessages, newMessages }) => {
+  const handleChange = (event) => {
+    setNewMessages(event.target.value);
+  };
+
+  return (
+    <form onSubmit={onFormSubmit}>
+      <h1 className="header-text">What's making you happy right now?</h1>
```

Comment:
This would rather be in a `label` linked to the `textarea`, especially for accessibility.